File: java/src/main/java/io/eventdriven/slimdownaggregates/book/entities/CommitteeApproval.java
package io.eventdriven.slimdownaggregates.book.entities;

public class CommitteeApproval {
  private final boolean isApproved;
  private final String feedback;

  public CommitteeApproval(boolean isApproved, String feedback) {
    this.isApproved = isApproved;
    this.feedback = feedback;
  }

  public boolean isApproved() {
    return isApproved;
  }

  public String getFeedback() {
    return feedback;
  }
}

File: csharp/Original/PublishingHouse.Persistence/Books/ValueObjects/CommitteeApproval.cs
namespace PublishingHouse.Persistence.Books.ValueObjects;

public class CommitteeApproval
{
    public CommitteeApproval(bool isApproved, string feedback)
    {
        IsApproved = isApproved;
        Feedback = feedback;
    }

    public bool IsApproved { get; }
    public string Feedback { get; }
}

